app/views/CheckRegisteredDetailsView.scala.html

```diff
@@ -14,22 +14,6 @@
  * limitations under the License.
  *@
```

Comment:
As discussed, please remove this view as it is unused and also clean up the message files, thanks.